### ISSUE TITLE
Mob chunks bug

### ISSUE THREAD

**Issue by Bosfik** [Divine-Journey-2/Divine-Journey-2#370]
**Modpack version:**
2.4.0

**Description:**
Mystical agriculture tier 1 mob chunks are only crafted from wolf's trophy

**Steps to reproduce:**
When I insert any other trophy, the recipe just doesn't work, although it shows(and should be obvious) that any trophy can be used

**Expected behavior:**
Let it craft from any trophy

**Comment by WaitingIdly** [comment]
where is this error occurring

**Comment by Bosfik** [comment]
In mekanism crusher and sag mill, pulverizer works fine

**Comment by Atricos** [comment]
I don't know what's causing the issue, I have provided the `*` tag (meaning any damage value can be used), and provided not NBT tags (meaning any NBT tag can be used). For now I disabled the recipes that weren't working (in 2.6.0: https://www.curseforge.com/minecraft/modpacks/divine-journey-2/files/3480373), but this isn't a complete fix. I'll have to figure out why the other crushers don't accept the trophies.
